game/database/domains/action/MELEE.json

```diff
@@ -11,17 +12,18 @@
       }],
     "params":[{
         "typename":"choose_target",
-        "max-range":1,
+        "body-only":{
+          "body-type":"<none>"
```

Comment:
I think this will crash when executed. But it's my bad, I'm the one who made input enums default to that `'<none>'` string.